README.md

```diff
@@ -28,8 +28,8 @@ provides functionality to perform statistical inference on estimated parameters.
 - The complete history of parameters and function evaluations can be saved in a database
   for maximum reproducibility.
 - Painless and efficient multistart optimization.
-- The progress of the optimization is displayed in real time via an interactive
-  dashboard.
+- The progress of the optimization is displayed in real time in `criterion_plots` and
+  `params_plots`.
 
```

Comment:
Not entirely convinced by this. You also say something similar elsewhere (see other comments).

My main problem with this is twofold:

(1) You do not mention that this requires logging (but to be fair, it was not mentioned beforehand), and
(2) It's not displayed in real-time unless you refresh the plotting manually (or am I missing something)

Reply:
I think for now it's the best thing we have so I wanted to point ex dashboard users to it.

Reply:
Makes sense. I do not have a strong opinion on this, but I would rather write something like this:

> The progress of the optimization can be displayed in real time using criterion and
> parameter plots.

I also just realized that you write `criterion_plots` instead of `criterion_plot`. This is slightly confusing since there is no `criterion_plots` function.

Reply:
Done.